pkgs/data/fonts/intel-one-mono/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenvNoCC, fetchurl, unzip}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "intel-one-mono";
+  version = "1.2.0";
+
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/otf.zip";
+      sha256 = "sha256-VnXIaW77dRXvXB1Vr01xRQDMECltwzF/RMqGgAWnu5M=";
+     })
+    (fetchurl {
+      url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/ttf.zip";
+      sha256 = "sha256-kaz0DePeO8nvKVonYJhs5f2+ps/5Xo5psjg1hoxzaiU=";
+    })
+  ];
+
+  # Multiple packages are download in srcs folder
+  # -> need to specify where the unpack phase will `cd` into
+  # use the `srcs` folder as the Root of the unpack phase
```

Comment:
```suggestion
```

Reply:
Here I would like to keep a message about this, because otherwise the unpackphase will not work correctly. And  IMO from the nixpkgs manual its not that easy to understand why this is necessary